src/Microsoft.AspNetCore.JsonPatch/Adapters/ObjectAdapter.cs

```diff
@@ -392,7 +441,7 @@ public void Copy(Operation operation, object objectToApplyTo)
             var visitor = new ObjectVisitor(parsedPath, ContractResolver);
 
             var target = objectToGetValueFrom;
-            if (!visitor.TryVisit(ref target, out var adapter, out var errorMessage))
+            if (!visitor.TryVisit(ref target, adapterWithTest: out var adapter, errorMessage: out var errorMessage))
```

Comment:
I'm not thrilled with the parameter name `adapterWithTest` - it's just verbose without communicating any more. You can also notice that it doesn't match the variable name `adapter`.